src/components/atoms/Todo.jsx

```diff
@@ -11,22 +11,35 @@ export default function Todo({ todo, toggleComplete, removeTodo }) {
     toggleComplete(todo.id);
   }
 
+  function handlePriority() {
+    setHighPriority(todo.id);
+  }
+
   return (
-    <ListItem style={{ display: "flex" }}>
+    <ListItem style={{ display: "flex", padding: 0 }}>
       <Checkbox
-        checked={todo && todo.completed}
```

Comment:
did you get this to work without checking for existence first?

Reply:
yes yes~~